webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -50,36 +52,51 @@
         </div>
       </div>
 
-      <!-- TODO: migrate and test once contact is done
       <div class="col-sm-4 general-actions left-pane" *ngIf="currentTab === 'contacts'">
-        <div class="actions dropup" mmAuth="" [[mmAuthAny]]="[ isAdmin, actionBar?.left?.childPlaces && 'can_create_places' ]">
-          <a class="mm-icon mm-icon-inverse mm-icon-caption dropdown-toggle" *ngIf="actionBar?.left?.childPlaces?.length > 1" data-toggle="dropdown" mmAuth="can_create_places">
+        <div class="actions dropup" mmAuth [mmAuthAny]="[ isAdmin, actionBar?.left?.childPlaces && 'can_create_places' ]">
+          <a *ngIf="actionBar?.left?.childPlaces?.length > 1"
+             mmAuth="can_create_places"
+             class="mm-icon mm-icon-inverse mm-icon-caption dropdown-toggle"
+             data-toggle="dropdown"
+          >
             <span class="fa fa-plus"></span>
             <p translate>Add place</p>
           </a>
-          <ul class="dropdown-menu mm-dropdown-menu with-icon" *ngIf="actionBar?.left?.childPlaces?.length > 1" mmAuth="can_create_places">
-            TODO: sort actionBar.left.childPlaces by "id" !!
-            <li *ngFor="let child of actionBar.left.childPlaces; trackBy: child.id">
-              <a ui-sref="contacts.addChild({ type: child.id, parent_id: actionBar.left.userFacilityId, from: 'list' })">
+
+          <ul *ngIf="actionBar?.left?.childPlaces?.length > 1"
+              mmAuth="can_create_places"
+              class="dropdown-menu mm-dropdown-menu with-icon"
+          >
+            <li *ngFor="let child of actionBar.left.childPlaces; trackBy: trackById">
+              <a (click)="navigateToContactAdd(child.id, actionBar.left.userFacilityId, 'list')">
                 <span [innerHTML]="child.icon | resourceIcon"></span>
                 <span class="content" translate>{{ child.create_key }}</span>
               </a>
             </li>
           </ul>
-          <a class="mm-icon mm-icon-inverse mm-icon-caption" *ngIf="actionBar?.left?.childPlaces?.length === 1" ui-sref="contacts.addChild({ type: actionBar.left.childPlaces[0].id, parent_id: actionBar.left.userFacilityId, from: 'list' })" mmAuth="can_create_places">
+
+          <a *ngIf="actionBar?.left?.childPlaces?.length === 1"
+             mmAuth="can_create_places"
+             class="mm-icon mm-icon-inverse mm-icon-caption"
+             (click)="navigateToContactAdd(actionBar.left.childPlaces[0]?.id, actionBar.left.userFacilityId, 'list')"
```

Comment:
Same as above, link instead of click event.

Reply:
Router link added